.eslintrc.js

```diff
@@ -39,4 +39,38 @@ module.exports = {
     semi: 2,
     strict: 0,
   },
+  settings: {
+    react: {
+      version: 'detect',
+    },
+  },
+  overrides: [
+    {
+      files: ['*.ts', '*.tsx'],
+      extends: ['plugin:@typescript-eslint/base'],
```

Comment:
if we're making a breaking change for TS anyways, and are adding a linter. I wanted to recommend we had the following config for boolean naming of variables:

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md#enforce-that-boolean-variables-are-prefixed-with-an-allowed-verb

This library in particular seems to have splitting behavior (`isClearable` / `meniIsOpen`)

Reply:
This seems like mostly an issue of style. Not saying it would be a bad change, but that that we would need to discuss this as a team (as with any breaking change). I try to stay out of stylistic issues as much as possible (since I think the disagreement over them are often not worth the possible benefit of a better style). My personal opinion is that the pain of everyone having to change their code would be worse than the consistency benefits in this case.

Reply:
Certainly. To your point, just because it's a breaking change... doesn't mean we want every little bit of the API broken.